### DISCUSSION TITLE
Ports to be open from firewall for epiphany to work

### DISCUSSION THREAD

**Discussion by rahulrajanrepo** [epiphany-platform/epiphany#3335] [General]
Hi All,

We have some restricted access with firewall and the machine is placed behind the firewall, that need to be run epiphany. Need to know what are all the ports that need to be open from firewall for epiphany to work.

**Comment by seriva** [comment]
Port 22 for SSH to work with Ansible and ports 6443, 80 for tunnels to Kubectl and helm I think. @przemyslavic anything more you can think of?

**Reply by przemyslavic** [reply]
This set of ports to be open seems to be sufficient.
@rahulrajanpondy Let us know if you run into any issues with this.

**Reply by rahulrajanrepo** [reply]
Sure, thanks for the update

**Reply by tomasz-baran** [reply]
I think for deployment time only 22 is needed since 6443 and 80 are tunneled via SSH.
